nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/StandardProvenanceReporter.java

```diff
@@ -226,7 +228,7 @@ public void send(final FlowFile flowFile, final String transitUri, final String
 
     @Override
     public void send(final FlowFile flowFile, final String transitUri, final boolean force) {
-        send(flowFile, transitUri, -1L, true);
+        send(flowFile, transitUri, -1L, force);
```

Comment:
This one I fixed while I was in there, the only thing that calls it is InvokeHttp which passes in true, so it hasn't been a bug, but it could be some day :P